### ISSUE TITLE
You can capture ships whose critical systems you've just plundered, rendering them unusable

### ISSUE THREAD

**Issue by Pointedstick** [endless-sky/endless-sky#4208]
**To Reproduce**
1. Disable a ship
2. Plunder its engines
3. Capture it

The ship is then stuck there forever. Something even worse happens if you then...

4. Land on any planet or station in the system with your flagship
5. Sell all plundered outfits on the Trading screen
6. Depart

You're told to buy engines for your ship in the outfitter before you can depart. If the planet or station you landed on has no outfitter and no shipyard, then your flagship cannot depart unless you disown the other ship! Players who don't know they can do this may be very confused and frustrated.

**Expected behavior**
Capturing a ship without engines, a reactor, a hyperdrive, etc. seems inherently problematic; maybe if you first plunder a ship's critical components and then capture it, they should be automatically re-installed so the ship is fully functional. Or maybe capturing and plundering should be made mutually exclusive such that if you first plunder stuff you shouldn't be able to then capture the ship.

**Comment by ghoulavenger** [comment]
I don't mind being locked out of capture after plundering, since you usually don't want to capture the ship anyway if you're plundering. But a failed capture should not be locked out of plundering. It would however be best if you were simply unable to capture a ship that wasn't space worthy (e.g. capturing triggers a preflight check and if it fails, you can't capture it).

**Comment by Wrzlprnft** [comment]
If i recall correctly, it's leftin the game on purpose?
I totally not recall the keyword here, but isn't there a way to give up the ship via fleet info screen for the second case?

**Comment by tehhowch** [comment]
"Disown"

Related: https://github.com/endless-sky/endless-sky/issues/2793#issuecomment-317109963

**Comment by jafdy** [comment]
Choosing when to lock it out would be hard, If you don't allow capture of ships without hyperdrive then what about fighters, or ships that were never meant to leave the system.
Likewise for reactors, perhaps it is a battery only ship. 

Automatic reinstalling is also problematic, people my have intentionally pulled that part out for some reason.

**Comment by Amazinite** [comment]
I honestly think this is just a case of "you messed up. Try again." When I first had this happen to me, it was more of a laughing moment than a frustrating one. Having it happen once or twice is also generally all that's needed before understanding why the ship you just captured isn't doing anything. I don't necessarily see the need to "fix" anything about it.

**Comment by Pointedstick** [comment]
Obviously a player who has gotten into this situation has messed up. :) The question is whether or not the game should let them mess up this way in the first place, either at all or without even a warning.

It also doesn't make logical sense to prevent the player's flagship from taking off from a planet when one of their escorts is missing critical components and therefore wouldn't have been able to land there with them.

**Comment by jafdy** [comment]
A warning wouldn't hurt.

**Comment by jostephd** [comment]
I agree that in this situation take-off shouldn't be blocked.

If this situation does happen, could we add another way to exit it? For example, let the player buy a set of engines into cargo, board his disabled escort, and install them. This would be especially useful if an escort ship had been disabled and plundered (but not destroyed) by AI ships, or it could become a gameplay mechanic for mission (`"A merchant ship was disabled and plundered in <waypoint>. Go there, give it engines/power/whatever it needs, and escort it to <destination>"`).

**Comment by ThrawnCA** [comment]
As discussed on #7735, one possible solution would be to apply validation when capturing a ship, similar to a preflight check, so if your crew beat the defenders but then discover that the ship isn't usable, they retreat instead of capturing.

**Comment by Zitchas** [comment]
I'd kind of like it to be a question, rather than an automatic. Maybe there's a planet in the system that has an outfitter, so I know I'll be able to re-outfit it or something?

So something more along the lines of, after capture, it does the preflight check, and if there's something that will stop the ship from functioning, the player gets a dialog box explaining that the ship won't be able to XYZ, do you want to proceed with capturing the ship?"

If a player absolutely wants to capture a ship, by all means, let them. I don't think we should hand-hold them. But giving them that information so they can make an informed decision and/or back out if they realize they made a mistake would be a good thing to do.

**Comment by ThrawnCA** [comment]
> If a player absolutely wants to capture a ship, by all means, let them. I don't think we should hand-hold them.

A ship with no engine shouldn't be able to land, though - the fact that it can is a bug - so boarding it should be a death sentence that the crew could quite reasonably refuse.

**Comment by xobes** [comment]
My initial reaction is: "don't do that" :-)

But it's worse than that.  It's checkmate for the player, isn't it?  No ability to move, land, etc.  Basically game-over due to other game mechanics.

**Comment by TomGoodIdea** [comment]
Why game over? Can't you just disown the ship?

**Comment by xobes** [comment]
Doh, my bad.  I was imagining that it became your ship -- then you'd be hosed.

**Comment by ThrawnCA** [comment]
Can you park a non-functional ship and get around the takeoff restrictions that way?

**Comment by TomGoodIdea** [comment]
Yes, but as you currently can't install outfits via boarding, there is no point in keeping a softlocked ship in your fleet (except maybe as a trophy).

**Comment by NobodyIII** [comment]
What's the status/plan on this? I have a stranded leviathan that I want to use. I looted its systems because I couldn't capture it, then I came back, and it was still there, because it was a bounty target. So I captured it, and now i have it parked (and impossible to unpark) in the ether, because it's in an uninhabited system with no planets to land on.

**Comment by Hecter94** [comment]
As far as I can tell, just reading through this issue and relevant discussions, there don't seem to be any changes to this system planned. 
If you loot essential systems off of a target, it becomes non-operative; capturing it while it's non-operative doesn't suddenly make it start working again. 
If you get into that situation, the solution is to disown the ship. 

I see there's some discussion about potentially adding a warning message in advance, which may be implemented. I can’t say for sure, but as far as I know, there's no plan to change the underlying mechanic.

**Comment by NobodyIII** [comment]
I should at least be able to loot the rest of the systems then. Also, how do I disown a ship?

**Comment by Amazinite** [comment]
In the player info panel (default `I`) while you're landed, there's a disown button when you're looking at one of your ships.

**Comment by warp-core** [comment]
> In the player info panel (default `I`) while you're landed, there's a disown button when you're looking at one of your ships.

You have to go to the ship info tab after selecting the ship, either from the tab at the top, the "s" key, or double clicking the ship in the list. Then the "disown" button will be at the bottom.
